AccountingSystem/Client/DataStoreClient.py

```diff
@@ -58,7 +55,13 @@ def addRegister( self, register ):
       return retVal
     if gConfig.getValue( '/LocalSite/DisableAccounting', False ):
       return S_OK()
+    
+    # a lock below can only prevent to send the register 
+    # with an 'earlier' commit what normally is not a problem at all
```

Comment:
For my understanding, can you just explain what you mean with "normally is not a problem at all"? Do you mean that we don't care if they are in different order, or do you mean that it should not happen even without the lock ?